torch_ttnn/patterns/addcdiv.py

```diff
@@ -0,0 +1,25 @@
+import torch
+
+
+import ttnn
+
+addcdiv = ttnn.operations.ternary.addcdiv
```

Comment:
it can as well be this
```suggestion
addcdiv = ttnn.addcdiv
```

My concern is that this copies a decorated object. Not sure what consequences this can have